castanet/counter.py

```diff
@@ -134,8 +156,16 @@ def assignment_count(cast_dict):
     return assignment_dictionary
 
 def aug_assigment_count(cast_dict):
-    """A function for counting the number of aug assignment"""
-    # An example of an aug assignmnet is x += 5
+    """A function for counting the number of aug assignment
```

Comment:
This should read `A function for counting the number of aug assignment statements in a Python program.`